lib/Optimizer/CodeGen/TranslateToOpenQASM.cpp

```diff
@@ -23,13 +23,12 @@ using namespace cudaq;
 //===----------------------------------------------------------------------===//
 // Helper functions
 //===----------------------------------------------------------------------===//
-
 /// Translates operation names into OpenQASM gate names
 static LogicalResult translateOperatorName(quake::OperatorInterface optor,
                                            StringRef &name) {
   StringRef qkeName = optor->getName().stripDialect();
   if (optor.getControls().size() == 0) {
-    name = StringSwitch<StringRef>(qkeName).Case("r1", "u1").Default(qkeName);
+    name = StringSwitch<StringRef>(qkeName).Default(qkeName);
```

Comment:
You don't need a `StringSwitch` if you are only using the default case.
```suggestion
    name = qkeName;
```

Reply:
restored the original version